weekly planner/styles.css

```diff
@@ -0,0 +1,113 @@
+:root {
+    --background-color: #f4f4f4;
+    --header-background-color: #f4f4f4;
+    --header-font-size: 1em;
+    --header-margin: 10px 0;
+    --calendar-padding: 10px 20px 20px 20px;
+    --calendar-gap: 10px;
+    --day-background-color: #fff;
+    --day-border-color: #ccc;
+    --day-padding: 10px;
+    --day-box-shadow: 2px 2px 5px #0000001a;
+    --header-font-weight: bold;
+    --header-border-bottom: 1px solid #ccc;
+    --header-margin-bottom: 10px;
+    --header-padding-bottom: 5px;
+    --event-title-font-weight: bold;
+    --event-description-margin-top: 5px;
+    --list-item-padding: 10px;
+    --list-item-border: 2px solid #ccc;
+    --list-item-border-radius: 5px;
+    --list-item-background-color: #b1dfec;
+    --nav-button-background-color: #007bff;
+    --nav-button-hover-background-color: #0056b3;
+    --nav-button-color: white;
+    --nav-button-padding: 10px;
+    --nav-button-font-size: 1em;
+    --nav-button-margin-left: 5px;
+    --nav-button-border-radius: 5px;
+}
+
+body {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+    background-color: var(--background-color);
+}
+
+.header-container {
+    display: flex;
+    align-items: center;
+    padding: 10px 20px;
+    background-color: var(--header-background-color);
+}
+
+.header {
```

Comment:
lots of selection by classes, its ok for now but it would be better to have distinct html elements